security/pkg/k8s/chiron/utils_test.go

```diff
@@ -397,8 +397,10 @@ func TestSubmitCSR(t *testing.T) {
 			cert.UsageClientAuth,
 		}
 
+		useV1 := true
```

Comment:
can you add a negative test case?

Reply:
Sure, I'll add a test case for this scenario.